docs/roles/developer/contracts/rpc.md

```diff
@@ -1,42 +0,0 @@
----
-id: rpc
-title: RPC
-sidebar_label: RPC
----
-
-The generic solution to invoking smart contracts on the NEAR Platform is our RPC interface. This method is used by `near-api-js`. Our [RPC interface](/docs/api/rpc) uses Standard JSON RPC 2.0 across the board.
```

Comment:
I think it makes sense to have this page under the developer role, so people can navigate through the menu while they are in the developer context. I would even argue that we need a similar page in the documentation for other roles as well for better discoverability.

P.S. the page don't need to provide the details (I think we want to move the diagram to the RPC documentation itself)